docs/server-request.md

```diff
@@ -6,7 +6,7 @@ sidebar_label: Server Request
 ​
 A low-level action that allows for any kind of HTTP request to be performed.
 
-## `Typical Use Cases`
+## `Typical use cases`
```

Comment:
Usually, titles are capitalized in English. Why do you feel that it should be lower-case?

Reply:
My mistake. Will fix.